pyproject.toml

```diff
@@ -40,6 +40,7 @@ classifiers = [
 ]
 dependencies = [
   "dask[array]",
+  # Release 0.3.1, needs new hyperspy release which includes https://github.com/hyperspy/hyperspy/pull/3494
```

Comment:
Should we wait for this to be done in this PR before merging?

Reply:
Yes, I agree: it is worth waiting for the hyperspy release to be available and do it straight after.

Reply:
Missed it here but updated in #108.